code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -188,6 +188,7 @@
 	new /obj/item/device/defib_kit/compact/combat/adv/loaded/cbo(src)
 	new /obj/item/oddity/chem_book(src)
 	new /obj/item/storage/hcases/med(src)
+	new /obj/item/storage/box/livesey(src) // Doctor Livesey. A very kind and merry person. Social personality type. Not married.
```

Comment:
Remove this. There will be no 'small change' for it. It's a costume and I don't want to place this in the head lockers.